src/shared/components/common/markdown-textarea.tsx

```diff
@@ -225,6 +230,8 @@ export class MarkdownTextArea extends Component<
                 >
                   <Icon icon="help-circle" classes="icon-inline" />
                 </a>
+                {this.props.maxLength !== undefined &&
```

Comment:
Can’t this use optional chaining? `this.props?.maxLength`

Reply:
Props cannot be undefined, so there is no need. However, having `!== undefined` was not necessary.